src/bioregistry/curation/curated_papers.csv

```diff
@@ -1 +1,21 @@
-pmid, relevant, relevancy_type, notes
\ No newline at end of file
+pmid,relevant,relevancy_type,notes
+39104285,1,new_provider,"Provider for UniProt IDs, issue with curation due to multiple URI formats depending on plant species"
+39074139,1,new_provider,"Resolver for PDB IDs"
+39014503,0,no_website,
+39047988,0,irrelevant_other,
+39115390,0,irrelevant_other,
+39095357,0,irrelevant_other,
+39084442,0,not_identifiers_resource,
+38991851,0,new_prefix,"identifiers for sharing, retrieving, and validating sample metadata."
+38991828,0,irrelevant_other,
+39049520,0,not_identifiers_resource,
+39104826,1,existing,"Already present in the bioregistry as a provider for mesh prefix"
+39050757,0,irrelevant_other,
+39064021,0,irrelevant_other,
+39028894,0,not_identifiers_resource,
+39044201,0,not_identifiers,"Potential resource for rare diseases identifiers, but not identifier information"
```

Comment:
Is the `relevancy_type` supposed to be `not_identifiers_resource` here?

Reply:
yes, 'not_identifiers_resource' is what I've been using. I must have accidentally deleted 'resource' when I was editing the file